week2-section1/exercise1/Main.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+* Exercise 1 - Understanding classes
+Create a class called FirstClass with a public constructor and destructor. Have the constructor output �I�m in the constructor!� and have the destructor output �I�m in the destructor!�. Then, instantiate 5 classes and run your program, are the constructor and destructor calls where you expected them to be?
+
+Now create a loop of at least five iterations. For each iteration, instantiate your class once. Run your program, are the constructor and destructor calls where you expected them to be?
+*/
+
+#include <iostream>
+
+class FMyFirtsClass
+{
+public:
+	FMyFirtsClass()
+	{
+		std::cout << "I'm in the constructor! " << std::endl;
+	};
+	~FMyFirtsClass()
+	{
+		std::cout << "I'm in the destructor! " << std::endl;
+	};
+};
+
+void InitializeClass() {
```

Comment:
Same thing with function names. This is not bad code, but the name is not precise. You are not _Initializing_ any class - you're instead creating instances of your class. So you could name this something more like `InstantiateClasses`, what do you think? Same as the next function.